opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseInteractor.java

```diff
@@ -265,6 +266,12 @@ public void run() {
                     properties.setParentId(operationalAreaId);
                     properties.setStatus(LocationProperty.PropertyStatus.PENDING_REVIEW);
                     properties.setUid(UUID.randomUUID().toString());
+                    String structureName = event.findObs(null, false, JsonForm.STRUCTURE_NAME).getValue().toString();
```

Comment:
There will be no observation if one does not provide this optional field. Also its not valid for mosquito and larval points. 

So calling to `toString` will result in NPE